packages/cli/src/config/PublishDetails.ts

```diff
@@ -125,19 +126,45 @@ export const loadPublishDetailsWithChecks = async (
   }
 
   config.release.media.forEach((item: PublishDetails["release"]["media"][0]) => {
-    const imagePath = path.join(process.cwd(), item.uri);
-    if (!fs.existsSync(imagePath) || !checkImageExtension(imagePath)) {
-      throw new Error(`Invalid media path or file type: ${item.uri}. Please ensure the file is a jpeg, png, or webp file.`);
+    const mediaPath = path.join(process.cwd(), item.uri);
+    if (!fs.existsSync(mediaPath)) {
+      throw new Error(`File doesnt exist: ${item.uri}.`)
+    }
+
+    if (item.purpose == "screenshot" && !checkImageExtension(mediaPath)) {
+      throw new Error(`Please ensure the file ${item.uri} is a jpeg, png, or webp file.`)
+    }
+
+    if (item.purpose == "video" && !checkVideoExtension(mediaPath)) {
+      throw new Error(`Please ensure the file ${item.uri} is a mp4.`)
     }
   }
   );
 
-  const previewMediaFiles = config.release.media?.filter(
-    (asset: any) => asset.purpose === "screenshot" || asset.purpose === "video"
+  const screenshots = config.release.media?.filter(
+    (asset: any) => asset.purpose === "screenshot"
+  )
+
+  for (const item of screenshots) {
+    const mediaPath = path.join(process.cwd(), item.uri);
+    if (await checkScreenshotSize(mediaPath)) {
+      throw new Error(`Screenshot ${mediaPath} must be at least 1080px in width and height.`);
+    }
+  }
+
+  const videos = config.release.media?.filter(
+    (asset: any) => asset.purpose === "video"
   )
 
-  if (previewMediaFiles.length < 4) {
-    throw new Error(`At least 4 screenshots or videos are required for publishing a new release. Found only ${previewMediaFiles.length}`)
+  for (const video of videos) {
+    const mediaPath = path.join(process.cwd(), video.uri);
+    if (await checkVideoSize(mediaPath)) {
+      throw new Error(`Video ${mediaPath} must be at least 1080px in width and height.`);
```

Comment:
That's a pretty high resolution for a video, given that the display on Saga is FHD+. Perhaps 720px would be a reasonable minimum resolution for videos?

Reply:
Reduced to 720p